rct_ros_tools/include/rct_ros_tools/print_utils.h

```diff
@@ -1,144 +1,21 @@
-{
-  std::cout << description << ":" << std::endl << transform.matrix() << std::endl << std::endl;
-  std::cout << "--- URDF Format " << parent_frame << " to " << child_frame << " ---" << std::endl;
-  Eigen::Vector3d rpy = transform.rotation().eulerAngles(2, 1, 0);
-  Eigen::Quaterniond q(transform.rotation());
-  std::cout << getStringXYZ(transform.translation()) << std::endl;
-  std::cout << getStringRPY(rpy) << std::endl;
-  std::cout << getStringQuaternion(q) << std::endl;
-}
-
-inline
-void printTransformDiff(const Eigen::Isometry3d &transform1, const Eigen::Isometry3d &transform2, const std::string &parent_frame, const std::string &child_frame, const std::string &description)
-{
-  Eigen::Isometry3d delta = transform1.inverse() * transform2;
-  Eigen::AngleAxisd aa (delta.linear());
-  Eigen::Vector3d rpy = delta.rotation().eulerAngles(2, 1, 0);
-
-  std::cout << description << ":" << std::endl;
-  std::cout << "--- " << parent_frame << " to " << child_frame << " Diff ---" << std::endl;
-
-  Eigen::Vector3d trans = transform2.translation() - transform1.translation();
-  std::cout << "DELTA S: " << trans.norm() << " at " << getStringXYZ(trans) << std::endl;
-  std::cout << "DELTA A: " << (180.0 * aa.angle() / M_PI) << " and " << getStringRPY(rpy) << std::endl << std::endl;
-
-  std::cout << "--- " << parent_frame << " to " << child_frame << " Diff Relative To Transform 1 ---" << std::endl;
-  std::cout << delta.matrix() << std::endl << std::endl;
-
-  std::cout << "DELTA S: " << delta.translation().norm() << " at " << getStringXYZ(delta.translation()) << std::endl;
-  std::cout << "DELTA A: " << (180.0 * aa.angle() / M_PI) << " and " << getStringRPY(rpy) << std::endl;
-}
-
-inline
-void printOptResults(bool converged, double initial_cost_per_obs, double final_cost_per_obs)
-{
-  std::cout << "Did converge?: " << converged << std::endl;
-  std::cout << "Initial cost?: " << std::sqrt(initial_cost_per_obs) << " (pixels per dot)" << std::endl;
-  std::cout << "Final cost?: " << std::sqrt(final_cost_per_obs) << " (pixels per dot)" << std::endl;
-}
-
-inline
-void printCameraIntrinsics(const std::array<double, 4> &values, const std::string &description)
-{
-  std::cout << description << ":" << std::endl;
-  std::cout << getStringIntrinsics(values) << std::endl;
-}
-
-inline
-void printCameraDistortion(const std::array<double, 5> &values, const std::string &description)
+namespace rct_ros_tools
 {
-  std::cout << description << ":" << std::endl;
-  std::cout << getStringDistortion(values) << std::endl;
-}
-
+using rct_common::getStringDistortion;
+using rct_common::getStringIntrinsics;
+using rct_common::getStringQuaternion;
+using rct_common::getStringRPY;
+using rct_common::getStringXYZ;
+using rct_common::printCameraDistortion;
+using rct_common::printCameraIntrinsics;
+using rct_common::printNewLine;
+using rct_common::printOptResults;
+using rct_common::printTitle;
+using rct_common::printTransform;
+using rct_common::printTransformDiff;
```

Comment:
This seems unnecessary; let's get rid of this file and just change references to this function to use the `rct_common` namespace

Reply:
This was to prevent breaking changes, but I am fine without removing.

Reply:
I removed this header and updated everywhere it was used. I add these changes the original commit so they are all in one commit.